config/fescar-config-core/src/main/java/com/alibaba/fescar/config/FileConfiguration.java

```diff
@@ -31,7 +31,7 @@
 
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigFactory;
-import org.apache.commons.lang3.ObjectUtils;
+import org.apache.commons.lang.ObjectUtils;
```

Comment:
Why not use lang3 ?

Reply:
lang3 is not a dependency here.

Reply:
Seata requires minimum JDK 1.8,so we can use lang instead of lang3,it will be ok.